src/parser/jobs/drg/Combos.js

```diff
@@ -0,0 +1,209 @@
+import {Trans, Plural, i18nMark} from '@lingui/react'
+import React, {Fragment} from 'react'
+import {Icon, Message} from 'semantic-ui-react'
+
+import {ActionLink} from 'components/ui/DbLink'
+import ACTIONS from 'data/ACTIONS'
+import CoreCombos from 'parser/core/modules/Combos'
+import {Rule, Requirement} from 'parser/core/modules/Checklist'
+import {TieredSuggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+
+const DRAGON_MAX_DURATION_MILLIS = 30000
+const DRAGON_DEFAULT_DURATION_MILLIS = 20000
+const BLOOD_EXTENSION_MILLIS = 10000
+
+const MAX_EYES = 3
+
+export default class Combos extends CoreCombos {
```

Comment:
Not sure how comfortable I am with combo handling and gauge tracking being rolled into one module, though I can see the benefit from the perspective of the combo requirements.